tests/unit/dstoreMemory/Test.js

```diff
@@ -93,22 +92,24 @@ define([
 						"element[3].innerHTML should be Selection 4");
 					assert.strictEqual(dstoreMemorylist1Elements[4].innerHTML, "Selection 5",
 						"element[4].innerHTML should be Selection 5");
-
-				});
+					d.resolve();
```

Comment:
It doesn't seem right to have both d.callback() and d.resolve().

I might have said the wrong thing before, the parameter to d.callback() must be a function, not a promise.   It can be a function that returns a promise though.